srcpkgs/pika-backup/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'pika-backup'
+pkgname=pika-backup
+version=0.7.4
+revision=1
+build_style=meson
+build_helper=rust
+hostmakedepends="cargo pkg-config gettext itstool"
+makedepends="rust-std libadwaita-devel gtk4-devel"
+checkdepends="gtk4"
```

Comment:
```sh
checkdepends="borg openssh xvfb-run"
make_check_pre="xvfb-run"
```

Reply:
why the openssh? just curios

Reply:
run the tests without it and see for yourself.